### PR TITLE
add new row in template

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1410
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/519
https://github.com/open-orchestra/open-orchestra-model-interface/pull/158

### CODE REVIEW

File: BackofficeBundle/Validator/Constraints/AreaFlexRowLayoutValidator.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    /**
+     * @param string $columnWidth
+     *
+     * @return bool
+     */
+    protected function validateColumnWidth($columnWidth)
+    {
+        $regex = "/^auto$|^\\d+px$|^\\d+%$|^\\d+$/";
```
Comment (nicolasThal) [inline BackofficeBundle/Validator/Constraints/AreaFlexRowLayoutValidator.php]: You could modify the regex with : `/^auto$|^\\d+(px|%)?$/`

**Comment by itkg-alavieille** [comment]
@nicolasThal PR updated

File: LogBundle/EventSubscriber/LogTemplateFlexSubscriber.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        $this->logger->info('open_orchestra_log.template_flex.area.delete', array(
+            'template_id' => $template->getTemplateId(),
+            'template_name' => $template->getName()
+        ));
+    }
+
+    /**
+     * @param TemplateEvent $event
+     */
+    public function templateAreaUpdate(TemplateEvent $event)
```
Comment (nicolasThal) [inline LogBundle/EventSubscriber/LogTemplateFlexSubscriber.php]: Are you sure about the `TemplateEvent` ?

**Comment by itkg-alavieille** [comment]
@nicolasThal method fix and add test for this method

File: BackofficeBundle/Tests/Validator/Constraints/AreaFlexRowLayoutValidatorTest.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            array('10%', 0),
+            array('100px', 0),
+            array('1/3', 1),
+            array('auto', 0),
+            array('10%,1/5', 1),
+            array('*', 1),
+            array('10%px', 1),
+            array('1/3px', 1),
+            array('10%, 100px, e', 1),
+            array('', 1),
```
Comment (nicolasThal) [inline BackofficeBundle/Tests/Validator/Constraints/AreaFlexRowLayoutValidatorTest.php]: You are forgetting a test case : '1' should pass I think

**Comment by nicolasThal** [comment]
:+1: with the new test case

**Pull request merged**